src/assets/assettypes.h

```diff
@@ -27,6 +27,13 @@ enum AssetType
     REISSUE
 };
 
+enum IssueAssetType
```

Comment:
Could these be moved somewhere into qt codebase since they only seem to apply there?